retesteth/configs/clientconfigs/oewrap.cpp

```diff
@@ -462,7 +469,8 @@ string const oewrap_package = R"(
     "keccak256": "^1.0.2",
     "yargs": "^13.2.4"
   }
-})";
+})
```

Comment:
are you sure this ) should be in here ? I should post BEGIN and END comments here where it is safe to edit